tests/framework/e2e/downgrade.go

```diff
@@ -56,7 +56,7 @@ func DowngradeCancel(t *testing.T, epc *EtcdProcessCluster) {
 	c := epc.Etcdctl()
 
 	var err error
-	testutils.ExecuteWithTimeout(t, 1*time.Minute, func() {
+	testutils.ExecuteWithTimeout(t, 2*time.Minute, func() {
```

Comment:
Can you elaborate why it takes so long to cancel the downgrade? @henrybear327 @siyuanfoundation

Reply:
I haven't had a solid conclusion so far. I will have to investigate deeper to be able to answer
1. How often are we getting the time out if we use 1 min
2. What are the usual errors

Reply:
@siyuanfoundation @henrybear327 can you take this as a high priority? I see multiple times of timeout of the downgrade e2e cases.

60s should be already long enough. We need a clearer understanding why it takes so long. 

also cc @fuweid

Reply:
I am checking this flaky case in my local. I think it maybe related to case. So, I think increasing timeout isn't working

Reply:
@fuweid Thanks for looking into this. Just assigned https://github.com/etcd-io/etcd/issues/19391 to you. Feel free to let me know if you need discussion tomorrow.

Reply:
I will keep looking into this tomorrow morning as my first task.